staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -1065,6 +1068,8 @@ message Image {
     // User name that will run the command(s). This is used if UID is not set
     // and no user is specified when creating container.
     string username = 6;
+    // ImageSpec for image which includes annotations
+    ImageSpec spec = 7;
```

Comment:
If we are embedding the entire `ImageSpec` here, which includes the image ID, should we remove the `id` field, as it seems redundant?

Reply:
This was discussed previously and it was recommended to duplicate the field here to ensure backwards compatibility. Let me try and find the relevant discussion.

Reply:
The last line in the section calls out the duplicated data 
https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/windows-runtimeclass-support.md#imagespec-as-part-of-the-image-struct

`Note that the ID field will be potentially duplicated in ImageSpec for backward compatibility.`